<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="PlotWidget" name="plotview"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="counter_enable_box_1">
       <property name="text">
        <string>Counter 1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="counter_enable_box_2">
       <property name="text">
        <string>Counter 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="counter_enable_box_3">
       <property name="text">
        <string>Counter 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="counter_enable_box_4">
       <property name="text">
        <string>Counter 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Integration time [ms]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="time_box">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="minimum">
        <double>10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Points to keep</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="remember_points_box">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DisplayUnit">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Counts</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
